src/Forms/SudoModePasswordField.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\Forms\FormField;
+
+class SudoModePasswordField extends FormField
+{
+    public const FIELD_NAME = 'SudoModePasswordField';
+    
+    protected $schemaComponent = 'SudoModePasswordField';
+
+    protected $extraClasses = [
+        'SudoModePasswordField',
+    ];
+
+    public function __construct($name, $title = null, $value = '')
+    {
+        // Name must be "SudoModePasswordField" as there's logic elsewhere expecting this
+        // $name and $value are set to null as the react component does not use these arguments
+        parent::__construct(SudoModePasswordField::FIELD_NAME, null, null);
```

Comment:
```suggestion
        parent::__construct(SudoModePasswordField::FIELD_NAME);
```
No need to be explicitly about passing `null` - those are already the default args.

Reply:
Updated